tests/e2e/utils.go

```diff
@@ -189,3 +204,140 @@ func JobIsCompleted(ctx context.Context, cli clientset.Interface, namespace, pod
 	}
 	return false
 }
+
+// randomSuffix provides a random sequence to append to pods,services,rcs.
+//
+//nolint:unused
```

Comment:
why not remove it, if unused?

Reply:
is unused by a non test pkg, so the linter goes crazy, we actually use the func on files with _test

Reply:
Why not move it to the test files? Would that satisfy the linter?

Reply:
done